added ff release note for exnref (#36131)
mdn/content: files/en-us/mozilla/firefox/releases/131/index.md

### WebAssembly

> - The new exception reference with the value of `exnref` can now be used to help with [Exception Handling](/en-US/docs/WebAssembly/JavaScript_interface/Exception), either from WASM to JavaScript or JavaScript to WASM. ([Firefox bug 1908375](https://bugzil.la/1908375)).

Reviewer: ```suggestion
- The new exception reference with the value of `exnref` can now be used to help with [exception handling](/en-US/docs/WebAssembly/JavaScript_interface/Exception) between WebAssembly and JavaScript in both directions. ([Firefox bug 1908375](https://bugzil.la/1908375)).
```
